cob_hardware_emulation/scripts/emulation_follow_joint_trajectory.py

```diff
@@ -62,16 +62,21 @@ def fjta_cb(self, goal):
             # This is used to compute the interpolation weight as a function of current and goal time point
             time_since_start_of_previous_point = rospy.Duration(0)
 
+            nr_points_dof = len(self.joint_names)
+
             # for all points in the desired trajectory
             for point in goal_sorted.trajectory.points:
 
+                if len(point.positions) != nr_points_dof:
```

Comment:
Added this check to ensure that the number of joint positions is always equal to the number of dofs (joint names count).
This was just identified by accident while looking through the code again, there was no issue resulting from that yet.